Makefile

```diff
@@ -205,7 +205,19 @@ test-e2e: e2e-setup
 			--up-local \
 			--go-test-flags "-timeout=15m" \
 			--local-operator-flags "$(ZAP_FLAGS)" \
-			$(OPERATOR_SDK_EXTRA_ARGS)
+			$(OPERATOR_SDK_EXTRA_ARGS) \
+			| tee test-e2e.log
+	$(Q)cat test-e2e.log | grep 'Local operator stderr:' | sed -e 's,.*Local operator stderr: \(.*\)}\\n",\1,g' \
```

Comment:
In command-line, I'm executing the following in order to render logs in a regular fashion: ```
make test-e2e |grep 'msg="Local operator stderr' |sed 's/^time.*err: //g' |eval 'in=$(cat); echo "$in"'$h
```
Would this also work in the `Makefile` context? Therefore we could avoid executing `sed` repeatedly.

Reply:
Ah, thanks for the tip ;)

Reply:
Unfortunately, I tried to add it in the Makefile but this doesn't work as-is (even with the `$` character escaped properly (with `$$`). The output still goes with escaped whitespace.

However, I've updated the Makefile according to your other suggestions.

Reply:
Looks good! Thanks :-)